stock_picking_package_preparation_line/models/stock_picking_package_preparation.py

```diff
@@ -171,8 +171,8 @@ def action_put_in_pack(self):
                 # preparation lines, updating lines would erase some fields
                 picking = picking.with_context(skip_update_line_ids=True)
                 picking.action_confirm()
-                # Show an error if a picking is not confirmed
-                if picking.state != 'confirmed':
+                # Show an error if a picking is not confirmed or assigned
+                if picking.state not in ('confirmed', 'assigned'):
```

Comment:
@eLBati As you changed this, you should include test.

Reply:
@rousseldenis what do you mean? That part is already covered

Reply:
Nope : https://codecov.io/gh/OCA/stock-logistics-workflow/compare/1e193d25dde1d9e23ed58bcd13335be02ab2a70c...79b5e82054339d234a06bf4613f573e08d731cc0/diff#D3-176

Reply:
@rousseldenis added a test and discovered that change what not needed actually. Thanks